src/saksoversikt/sakpanel/SakPanel.tsx

```diff
@@ -48,7 +48,7 @@ const SakPanel: React.FC<Props> = ({fiksDigisosId, tittel, oppdatert, url, kilde
 
     const oppdatertTittel = useMemo(() => {
         if (saksdetaljer && saksdetaljer.soknadTittel?.length > 0) {
-            return saksdetaljer.soknadTittel;
+            return saksdetaljer.soknadTittel.replaceAll("saker.default_tittel", t("saker.default_tittel"));
```

Comment:
Ah, du gjør det ja :sweat_smile: Hvis du endrer til 
```suggestion
            return saksdetaljer.soknadTittel.replaceAll("default_sak_tittel", t("saker.default_tittel"));
```
her, så slipper du å oppdatere innsyn-api (tror jeg?)

Reply:
Det blir så mye bedre om jeg gjør det😅, tenkte ikke på at jeg kunne faktisk gjør det sånt😶